packages/service/ni_measurement_plugin_sdk_service/_internal/grpc_servicer.py

```diff
@@ -181,6 +182,8 @@ def __init__(
         self._measure_function = measure_function
         self._owner = weakref.ref(owner) if owner is not None else None  # avoid reference cycle
         self._service_info = service_info
+        self._configuration_parameters_message_type = service_info.service_class + ".Configurations"
```

Comment:
We should think through what happens with multiple versions of the same service class, and potentially write some tests. This can be deferred to a separate PR / work item though.

Reply:
If I just directly check the type_url of the received object, is that just as good as calling `Is`? Because that would remove any issues with conflicts when trying to instantiate a message via the message_factory. Though I suppose there might still be some problems when trying to load the measurement in the first place (wherein it loads its types into the descriptor_pool).

Reply:
Yes, it's fine. At this point, I'm convinced that it's a feature for the client to insulate itself & the descriptor pool from the service's advertised message names.